ibis/tests/all/test_aggregation.py

```diff
@@ -3,10 +3,24 @@
 import pytest
 from pytest import param
 
+import ibis.expr.datatypes as dt
 from ibis.tests.backends import Clickhouse, MySQL, Postgres, PySpark, SQLite
+from ibis.udf.vectorized import reduction
+
+
+@reduction(input_type=[dt.double], output_type=dt.double)
+def mean_udf(s):
+    return s.mean()
+
 
 aggregate_test_params = [
     param(lambda t: t.double_col.mean(), lambda s: s.mean(), id='mean',),
+    param(
```

Comment:
This is tested with `table.groupby(..).aggregate(..)`?

Reply:
Yes these `aggregate_test_params` are used by these two tests:
- [`test_aggregate`](https://github.com/ibis-project/ibis/blob/d679815b4a86272b0ef7e3eed8e4216d735a701e/ibis/tests/all/test_aggregation.py#L38) (tests `table.aggregate(...)`)
- [`test_aggregate_grouped`](https://github.com/ibis-project/ibis/blob/d679815b4a86272b0ef7e3eed8e4216d735a701e/ibis/tests/all/test_aggregation.py#L53) (tests `table.groupby(...).aggregate(...)` and `table.aggregate(..., by=...)`)

Reply:
Sounds good